src/server/game/Entities/Unit/StatSystem.cpp

```diff
@@ -1075,10 +1075,10 @@ void Creature::CalculateMinMaxDamage(WeaponAttackType attType, bool normalized,
         weaponMaxDamage = 0.0f;
     }
 
-    // pussywizard: subtract value from database till its fixed (the way it worked before creature_levelstats damage implementation)
-    float attackPower      = GetTotalAttackPowerValue(attType) - (attType == RANGED_ATTACK ? GetCreatureTemplate()->rangedattackpower : GetCreatureTemplate()->attackpower);
-    float baseValue        = GetModifierValue(unitMod, BASE_VALUE) + (attackPower * GetAPMultiplier(attType, normalized) / 14.0f);
-    float basePct          = GetModifierValue(unitMod, BASE_PCT);
+    float attackPower      = GetTotalAttackPowerValue(attType);
+    float attackSpeedMulti = GetAPMultiplier(attType, normalized);
```

Comment:
What is normalized doing here?

Reply:
In this case nothing. Its just that GetAPMultiplier expects this bool, and we get it from CalculateMinMaxDamage(). The function was called like that before, its just that it was calculated at the wrong spot.

`if (!normalized || GetTypeId() != TYPEID_PLAYER)
        return float(GetAttackTime(attType)) / 1000.0f;`

Thats how GetAPMultiplier calculates the modifier, so it doesnt matter in this case if normalized is true or false as we aren't a player.

I think this function is for something else when we use GetAPMultiplier on a player instead of a creature.